packages/jest-cli/src/run_jest.js

```diff
@@ -77,7 +77,7 @@ const processResults = (runResults, options) => {
   return options.onComplete && options.onComplete(runResults);
 };
 
-export default async function runJest({
+export default (async function runJest({
```

Comment:
why this paren? (weird that prettier doesn't normalize either way)

Reply:
Hah, apparently I wondered the same thing 6 months ago.

https://github.com/prettier/prettier/pull/1047#issuecomment-294939959

Reply:
it is fixed in babel but not in [flow](https://github.com/facebook/flow/issues/3264)
[prettier playground with flow](https://prettier.io/playground/#%7B%22content%22%3A%22export%20default%20async%20function%20bar()%20%7B%7D%22%2C%22options%22%3A%7B%22ast%22%3Afalse%2C%22bracketSpacing%22%3Atrue%2C%22doc%22%3Afalse%2C%22jsxBracketSameLine%22%3Afalse%2C%22output2%22%3Afalse%2C%22parser%22%3A%22flow%22%2C%22printWidth%22%3A80%2C%22semi%22%3Atrue%2C%22singleQuote%22%3Afalse%2C%22tabWidth%22%3A2%2C%22trailingComma%22%3A%22none%22%2C%22useTabs%22%3Afalse%7D%7D)